static/docs/user-guide/install/dev.md

```diff
@@ -0,0 +1,45 @@
+# Install for testing and development
+
+If you want to test the latest development version of DVC, you can install it
+from GitHub. In this case it is highly recommended to use
+[virtualenv](https://virtualenv.pypa.io/en/stable/), in order to encapsulate
+your testing environment.
+
+You can do it like this:
+
+```dvc
+$ virtualenv -p python3 .env
+$ source .env/bin/activate
+$ pip install gitpython
+$ pip install git+https://github.com/iterative/dvc.git
+```
+
+> **Note**! `gitpython` should be installed first to allow `setup.py` to
+> dynamically generate dvc version from the current git commit SHA. It allows us
```

Comment:
I'm not getting why recommend this `gitpython` method instead of the existing regular dev install method from https://dvc.org/doc/user-guide/contributing#development-environment.

Reply:
Also, please capitalize DVC or use backquotes (`dvc`) if referring to the command.
And please use _articles_: "**a** DVC version", "**the** official DVC release" 🙂

Reply:
> I'm not getting why recommend this gitpython method instead of the existing regular dev install method from https://dvc.org/doc/user-guide/contributing#development-environment.

I don't know, I tried to consolidate this page: https://dvc-org-pr-695.herokuapp.com/doc/user-guide/development
which is actually linked from here: https://dvc-org-pr-695.herokuapp.com/doc/get-started/install

Reply:
Woah! I never noticed that content was duplicated before (even when I've participated in the file's [history](https://github.com/iterative/dvc.org/commits/master/static/docs/user-guide/development.md) 😆 ! Thanks for pointing this out... I'm opening a separate issue to address this: #700